memo_db.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class MemoDB
+  CONN = PG.connect(dbname: 'memo_db')
+
+  class << self
+    def read
+      memos = CONN.exec('SELECT * FROM memos ORDER BY id')
+      memos.map do |memo|
+        [memo['id'], { 'title' => memo['title'], 'body' => memo['body'] }]
+      end.to_h
```

Comment:
```suggestion
      CONN.exec('SELECT * FROM memos ORDER BY id')
```

main.rbやerbの修正が必要になるかもしれませんが、execの実行結果をそのまま返してもいいんじゃないかなと思いました。
今はidとtitleとbodyしかないけど、項目が増えれば増えるほどデータの移し替え処理が増えてしまうので。

Reply:
to_hよりもさらに処理がシンプルになりました！erbファイルの修正もほんの少しでした。元のデータを加工して返すと融通が効かなくなることもあるんですね。